kubernetes/helm_charts/core/inquiryportal/templates/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+data:
+  {{- range $key, $val := .Values.knowlgbbportalenv }}
```

Comment:
Please update this to Values.inquiryportalenv @pallakartheekreddy

Reply:
@G33tha updated

Reply:
@G33tha updated